pkgs/by-name/je/jellyfin-tui/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  mpv,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "jellyfin-tui";
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "dhonus";
+    repo = "jellyfin-tui";
+    tag = lib.strings.concatStrings [
+      "v"
+      version
+    ];
```

```suggestion
    tag = "v${version}";
```
Use string interpolation. It is much more concise.

---

Not really sure how this didn't occur to me, gonna chalk it up to a tired brainfart last night lol
Thanks for being patient with me